test/src/job_manager.py

```diff
@@ -254,6 +256,37 @@ def ListFilesInDir(folder: str, ext=None):
     return files
 
 
```

Comment:
isn't flake8 complaining about extra empty line? @andrsd is python linting for the entire code or only some folders?

Reply:
1. By the coding standard enforced by flake8, free functions must be separated by 2 empty lines, methods (member functions) by 1 empty line
2. linting is enforced on the [whole code](https://github.com/Open-Sn/opensn/blob/d8db2cee927e4b4842e1468233aa6b74966a2864/.github/workflows/qa.yaml#L59-L61)